_config.yml

```diff
@@ -128,6 +128,15 @@ exclude:
   - tsconfig.json
   - vendor
   - webpack.config.js
+
+copy_files:
+  - from:
+      - node_modules/@18f/identity-design-system/dist/assets/css
+      - node_modules/@18f/identity-design-system/dist/assets/fonts
```

Comment:
I forgot to include `img` here, which is a little concerning that the build passed 😬 

I'll see if I can get a similar spec to what we have for checking console errors in the IdP.

Reply:
> I forgot to include `img` here, which is a little concerning that the build passed 😬
> 
> I'll see if I can get a similar spec to what we have for checking console errors in the IdP, which might have helped.

Spec added in 949e9c4506605254fee1f4795759ccd06f7a6747, fixed in a096da537facd6ae58e300ae4a6046fd34ac14e6.